resources/test_fonts/rebuild.sh

```diff
@@ -30,7 +30,11 @@ echo "Installing fonttools and freetype-py"
 $PIP install --upgrade pip
 $PIP install -r $REQUIREMENTS
 
+source $VENV_DIR/bin/activate
+
 for f in $(ls $SRC_DIR/*.ttx); do
     $TTX -o $OUT_DIR/$(basename "$f" .ttx).ttf --no-recalc-timestamp -b $f
     python3 $EXTRACT_GLYPHS $OUT_DIR/$(basename "$f" .ttx).ttf
```

Comment:
@cmyr @dfrg  if instead of calling `python3` you had called `$VENV_DIR/bin/python` then you would not need to first source the activate script